blackbox/docs/sql/general/lexical-structure.rst

```diff
@@ -39,6 +39,40 @@ single quotes, e.g. ``'Jack''s car'``.
    Two adjacent single quotes are **not** equivalent to the double-quote
    character ``"``.
 
+.. _sql_escape_string_literals:
+
+String Literals with C-Style Escapes
+------------------------------------
+
+In addition to the escaped character ``'``, CrateDB supports C-Style escaped
+string sequences. Such a sequence is constructed by prefixing the string
+literal with the letter ``E`` or ``e``, for example, ``e'hello\nWorld'``.
+The following escaped sequences are supported:
+
+==================================================   ================
```

Comment:
Is this formatted correctly as table in sphinx ?

Reply:
Yes, I used the [simple table](http://docutils.sourceforge.net/docs/user/rst/quickref.html#tables) syntax

see 
<img width="926" alt="image" src="https://user-images.githubusercontent.com/41899848/53347016-e974e680-3920-11e9-8b7f-9ffb55b68e05.png">